modules/libcom/src/iocsh/iocsh.h

```diff
@@ -86,6 +86,20 @@ LIBCOM_API const iocshVarDef * epicsStdCall iocshFindVariable(
 /* This should only be called when iocsh is no longer needed*/
 LIBCOM_API void epicsStdCall iocshFree(void);
 
+/** Helper for implementing main() for an IOC.
+ *
+ * @code
+ * int main(int argc,char *argv[]) {
+ *     return iocshMain(argc, argv);
+ * }
+ * @endcode
+ *
+ * see the 'ioc' template of makeBaseApp.pl for an example with compatibility
+ * for older Base.
+ *
+ * @since UNRELEASED
```

Comment:
@anjohnson Something I've started doing with [PVXS](https://mdavidsaver.github.io/pvxs/details.html#contributing) which I'd like to start doing with the 7.0 branch as well.  The idea is to later do a search+replace of `UNRELEASED` with the actual version number just prior cutting a release.